gulpfile.js

```diff
@@ -180,16 +181,20 @@ function build() {
       ghPagesBuild()
     ),
     parallel(npmCss, npmStats, npmPackage, npmSrc, npmBrowser),
+    cleanUpVersions,
     linkLatest
   );
 }
 
-function _replaceVersionInFile(filename) {
-  var updatedFile = fs.readFileSync(filename, 'utf-8').replace(/@VERSION/g, VERSION);
-
-  fs.writeFileSync(filename, updatedFile, 'utf-8');
+function cleanUpVersions() {
+  return src('dist/**/*.{js,html}')
+    .pipe(replace('{@pkg version}', VERSION))
+    .pipe(replace('{@pkg bt-web-version}', BT_WEB_VERSION))
```

Comment:
If we're now doing this here, should we remove this plugin? https://github.com/braintree/braintree-web-drop-in/blob/c8bc81b64def5ef739ea5b42db0c65c7d1fc75ce/jsdoc/version-interpolator-plugin.js

Reply:
No, because when we do, jsdoc no longer takes markdown links and turns them into anchor tags